src/main/java/lotto/domain/lotto/LottoTicket.java

```diff
@@ -3,43 +3,32 @@
 import java.util.Arrays;
 import java.util.HashSet;
 import java.util.List;
-import java.util.Set;
 import java.util.stream.Collectors;
-import lotto.domain.game.LottoMachine;
-import lotto.domain.game.Rank;
 
 public class LottoTicket {
+
   private static final int LOTTO_CNT = 6;
 
   private final List<LottoNumber> numbers;
 
-  private LottoTicket(List<LottoNumber> numbers) {
+  public LottoTicket(List<LottoNumber> numbers) {
     validate(numbers);
     this.numbers = List.copyOf(numbers);
   }
 
-  public static LottoTicket of(List<LottoNumber> numbers) {
-    return new LottoTicket(numbers);
-  }
-
-  public static LottoTicket from(String numbers) {
-    return new LottoTicket(convert(parse(numbers)));
+  public LottoTicket(String numbers) {
+    this(convert(parse(numbers)));
   }
 
-  public static LottoTicket of(Integer... numbers) {
-    return new LottoTicket(convert(List.of(numbers)));
+  public LottoTicket (Integer... numbers) {
+    this(convert(List.of(numbers)));
   }
```

Comment:
```suggestion
public LottoTicket(String numbers) {
    this(convert(parse(numbers)));
}

  public LottoTicket (Integer... numbers) {
    this(convert(List.of(numbers)));
}

  public LottoTicket(List<LottoNumber> numbers) {
validate(numbers);
this.numbers = List.copyOf(numbers);
}
```

필드 값을 초기화하는 주 생성자를 마지막에 구현하는 것이 관례